src/content/docs/en/guides/cms/gitcms.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: GitCMS & Astro
+description: Integrate GitCMS into your Astro project for seamless content management
+sidebar:
+  label: GitCMS
+type: cms
+service: GitCMS
+i18nReady: true
+---
+
+[GitCMS](https://gitcms.blog) is a Chrome extension that bridges the gap in static site generation (SSG) workflows by enhancing the writing and management experience for markdown files. It transforms GitHub into a powerful headless CMS, providing a Notion-like editing experience directly within GitHub's interface. With GitCMS, developers, clients, and content writers can write, manage, and search markdown content effortlessly — eliminating the pain of editing raw markdown or juggling file management. This makes managing content for Astro sites smoother, without additional servers or complex CMS setups.
+
+## Resources
+- [Introducing GitCMS](https://gitcms.blog/posts/introducing-gitcms/)
+- [How to Configure GitCMS](https://gitcms.blog/posts/how-to-configure-gitcms/)
+- [Frontmatter Fields in GitCMS](https://gitcms.blog/posts/frontmatter-editor/)
```

Comment:
```suggestion
```
We don't typically include multiple resources from the same official source. We don't need a link to each, individual topic. A link to overall documentation (or, in your case an introduction) and to the Chrome extension in the web store is fine! People can find what they need!